modules/ScreenNameLocalFormatter/src/main/java/com/liferay/formatter/service/NewUserLocalServiceWrapper.java

```diff
@@ -0,0 +1,384 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.formatter.service;
+
+import com.liferay.formatter.keys.NewFormatterKeys;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.service.ServiceWrapper;
+import com.liferay.portal.kernel.service.UserLocalService;
+import com.liferay.portal.kernel.service.UserLocalServiceWrapper;
+import com.liferay.portal.kernel.util.Props;
+import com.liferay.portal.kernel.util.Validator;
+
+import java.util.Locale;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author leonardo.ferreira
+ */
+@Component(immediate = true, property = {}, service = ServiceWrapper.class)
+public class NewUserLocalServiceWrapper extends UserLocalServiceWrapper {
+
+	public NewUserLocalServiceWrapper() {
+		super(null);
+	}
+
+	/**
+	 * Adds a default admin user for the company.
+	 *
+	 * @param companyId    the primary key of the user's company
+	 * @param screenName   the user's screen name
+	 * @param emailAddress the user's email address
+	 * @param locale       the user's locale
+	 * @param firstName    the user's first name
+	 * @param middleName   the user's middle name
+	 * @param lastName     the user's last name
+	 * @return the new default admin user
+	 */
+	@Override
+	public User addDefaultAdminUser(
+			long companyId, String screenName, String emailAddress,
+			Locale locale, String firstName, String middleName, String lastName)
+		throws PortalException {
+
+		String screenNamewithEmail = addEmailIfIsNotEmail(screenName);
```

Comment:
Typo.
screenName**w**ithEmail

W should be uppercase

Reply:
Fixed. Thanks.